scripts/scil_script_rename.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
We are nearly done renaming all scripts. I'm not sure this should exist in scilpy. But it's impressive that you did it. Or then, hide this outside from the scripts folder so that it doesn't get used by anyone outside the lab.

Reply:
I'll stick this in my private toolbox :stuck_out_tongue: